packages/realm/react-native.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
   dependency: {
     platforms: {
       android: {
-        sourceDir: "react-native/android",
+        sourceDir: "binding/android",
```

Comment:
I choose to move the `react-native/ios` and `react-native/android` into `binding` to co-locate it with all the other binding source-code.